github-actions/feature-request/src/octokit.ts

```diff
@@ -77,39 +79,37 @@ export class OctoKit implements GitHubAPI {
       return this._orgMembers.has(name);
     }
 
-    const response = this.octokit.paginate.iterator(
-      this.octokit.orgs.listMembers.endpoint.merge({
-        org,
-        per_page: 100,
-      }),
-    );
+    const response = this.octokit.paginate.iterator(this.octokit.orgs.listMembers, {
+      org,
+      per_page: 100,
+    });
 
     for await (const page of response) {
-      for (const user of page.data as Octokit.OrgsListMembersResponse) {
-        this._orgMembers.add(user.login);
+      for (const user of page.data) {
+        this._orgMembers.add(user!.login);
```

Comment:
In what case is `user` null/undefined?

Reply:
In the typings evidently, but from the results that are actually made its not.

Reply:
okay, definitely surprising because then a page response would have an array with a `undefined` or `null` value inside. so yeah, probably an issue with types.